src/test/java/trello/steps/CardSteps.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package trello.steps;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import org.testng.Assert;
+import trello.entities.Context;
+import trello.ui.pages.BoardPage;
+import trello.ui.pages.HomePage;
+
+/**
+ * CardSteps class.
+ *
+ * @author Josue Rodrguiguez.
+ * @version 0.0.1
+ */
+public class CardSteps {
+    private HomePage homePage;
```

Comment:
Please, add a white line above this line.

Reply:
Solved